gameapplication.py

```diff
@@ -0,0 +1,99 @@
+import time
+import re
+import IdentifyBaseWord as np
+import wordlist as wl
+
+#gameState 0 means the game is not being played
+gameState = 0
+
+
+print("Hello welcome to MediaTek's Spelling bee!")
+print("The goal of this game is to guess as many words as you can to accumulate points!")
+playGame = input("Would you like to play our game? enter yes or no: ")
+
+while (playGame != 'yes') and (playGame != 'no'):
+    print("Invalid input please enter yes or no!")
+    playGame = input("Would you like to play our game? enter yes or no\n")
+    if(playGame == "yes" or playGame == "no"):
+        break
+
+if(playGame == "no"):
+    print("Ok, goodbye!")
+    time.sleep(2.5)
+    exit()
+if(playGame == "yes"):
+    gameState = 1
+
+while(gameState == 1):
+    print("Commands should always we started with -, an example would be -help")
+    print("All input without - are considered guesses")
```

Comment:
Something that bothers me with the CLI is how it can be hard to differentiate visually between all the lines and my input, which is something to keep in mind when designing our messages!